webhook/logpipeline/v1beta1/defaulter.go

```diff
@@ -12,25 +12,25 @@ import (
 )
 
 // +kubebuilder:object:generate=false
-var _ webhook.CustomDefaulter = &LogPipelineDefaulter{}
+var _ webhook.CustomDefaulter = &defaulter{}
 
-type LogPipelineDefaulter struct {
+type defaulter struct {
 	RuntimeInputEnabled          bool
 	RuntimeInputKeepOriginalBody bool
 	DefaultOTLPOutputProtocol    telemetryv1beta1.OTLPProtocol
 }
 
-func SetupLogPipelineWebhookWithManager(mgr ctrl.Manager) error {
+func SetupWithManager(mgr ctrl.Manager) error {
```

Comment:
Probably you also want to move this func into a `setup.go` file, right?

Reply:
Yes, you are right